dbld/builddeps

```diff
@@ -193,6 +193,11 @@ function install_pip_packages {
                 pip install --upgrade setuptools
                 filter_packages_by_platform $DBLD_DIR/pip_packages.manifest | xargs pip install --ignore-installed -U
                 ;;
+            centos-7|ubuntu-xenial)
```

Comment:
Basically I'm okay with the fix, but I wonder, if it's not easier to switch problematic platforms to python3 instead?
I know `kira` cannot be moved, but centos-7 and xenial could be.

I've just experimented with it and it works. Unfortunately, you have to edit 3 different files to move a platform from python2 to python3:
```diff
$ git diff
diff --git a/dbld/build.manifest b/dbld/build.manifest
index 93d73afe2..ff6a689fb 100644
--- a/dbld/build.manifest
+++ b/dbld/build.manifest
@@ -38,5 +38,5 @@ ubuntu-bionic         python2
 ubuntu-focal           python3
 
 centos-6               python2,nokafka,nomongodb               CFLAGS=-D__be16=guint16,PYTHON_LIBS=-lpython2.6,PYTHON_CFLAGS=-I/usr/include/python2.6
-centos-7               python2,nokafka
+centos-7               python3,nokafka
 fedora                 python3
diff --git a/dbld/builddeps b/dbld/builddeps
index 2cfd946de..7a8571a5f 100755
--- a/dbld/builddeps
+++ b/dbld/builddeps
@@ -176,7 +176,7 @@ function install_pip_packages {
         fedora-32)
             python_executables="python"
             ;;
-        ubuntu-focal|tarball)
+        centos-7|ubuntu-focal|tarball)
             python_executables="python3"
             ;;
         devshell|kira)
diff --git a/dbld/packages.manifest b/dbld/packages.manifest
index be0750838..5a9df416b 100644
--- a/dbld/packages.manifest
+++ b/dbld/packages.manifest
@@ -88,12 +88,12 @@ automake                    [centos, fedora]
 
 python-dev                  [debian-stretch, debian-buster, ubuntu-xenial, ubuntu-bionic]
 python2-dev                 [debian-sid]
-python-devel                [centos, fedora]
-python3-devel               [fedora]
+python-devel                [centos-6, fedora]
+python3-devel               [fedora, centos-7]
 python-unversioned-command  [fedora]
 python3-dev                 [ubuntu-focal]
-python-pip                      [centos, fedora, debian-stretch, debian-buster, ubuntu-trusty, ubuntu-xenial, ubuntu-bionic]
-python3-pip                     [debian, ubuntu]
+python-pip                      [centos-6, fedora, debian-stretch, debian-buster, ubuntu-trusty, ubuntu-xenial, ubuntu-bionic]
+python3-pip                     [debian, ubuntu, centos-7]
 
 # libmongoc and libbson packages on various platforms
 # Because we are using fixed version of libmongoc on Bionic, we need to

```